presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -283,4 +287,32 @@ public QueryManagerConfig setQueryExecutionPolicy(String queryExecutionPolicy)
         this.queryExecutionPolicy = queryExecutionPolicy;
         return this;
     }
+
+    @Min(1)
+    public int getInitialMinWorkersRequired()
+    {
+        return initialMinWorkersRequired;
+    }
+
+    @Config("query-manager.initial-min-workers-required")
+    @ConfigDescription("Minimum number of workers that must announce themselves before the cluster is ready to accept queries")
```

Comment:
"Minimum number of workers that must be available before the cluster will accept queries"